postmortems/2024-03-12-zktrie-hardfork.md

```diff
@@ -0,0 +1,40 @@
+# 2024-03-12 ZKTrie Hardfork Post-Mortem
+
+# Incident Summary
+
+On March 12, 2024, at 15:16:25 UTC, a hardfork occurred at block number 8171899.
```

Comment:
I guess 'fork' is the more appropriate word than 'hardfork' in this case.

Reply:
Changed to 'fork' as you mentioned. Thank you